### PR TITLE
update rero.json

### PR DESCRIPTION
**Pull request by grolimur**
added fields including "_icon"

### CODE REVIEW

**Comment by egh** [comment]
@grolimur You don't actually need all of this, see https://github.com/egh/zotero-lookup-engines/blob/master/engines_json/epfl.json for all that you need. The JSON files generated for `engines.json` have a lot of boilerplate you don't need.

**Comment by grolimur** [comment]
Ok. Fine.
Can the favicon be added in order to find the right engine easier in the list?

**Comment by egh** [comment]
Yes, good idea! done 8e90279f

**Comment by egh** [comment]
If you have other ideas for how to improve the list, please let me know! I know it's pretty bad right now.

**Comment by grolimur** [comment]
Thanks.
I'm digging into this and will let you know if I have other ideas. :-)

File: engines_json/rero.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
   "title": "RERO",
-  "linktemplate": "http://resolver.rero.ch/rero?{z:openURL}"
-}
\ No newline at end of file
+  "_icon": "https://www.rero.ch/favicon.ico",
```
Comment (egh) [inline engines_json/rero.json:3]: This should just be `icon`.

Comment (grolimur) [reply engines_json/rero.json]: Changed

**Comment by egh** [comment]
Looks good, thanks!

**Comment by egh** [comment]
http://egh.github.io/zotero-lookup-engines/rero.html

**Pull request merged**